### ISSUE TITLE
Multiple blocks render incorrectly after the first block

### ISSUE THREAD

**Issue by rcrowe** [abhinav/goldmark-mermaid#3]
When multiple mermaid code blocks exist in the first file, after the first one they are rendered with `<pre><code class="language-mermaid">`, so are not rendered.

```
    testutil.go:176: ============= case 1 ================
        Markdown:
        -----------
        Transforms mermaid blocks.

        ```mermaid
        graph TD;
            A-->B;
            A-->C;
            B-->D;
            C-->D;
        ```

        ```mermaid
        graph TD;
            A-->B;
            A-->C;
            B-->D;
            C-->D;
        ```

        Expected:
        ----------
        <p>Transforms mermaid blocks.</p>
        <div class="mermaid">graph TD;
            A--&gt;B;
            A--&gt;C;
            B--&gt;D;
            C--&gt;D;
        </div><div class="mermaid">graph TD;
            A--&gt;B;
            A--&gt;C;
            B--&gt;D;
            C--&gt;D;
        </div><script src="mermaid.js"></script><script>mermaid.initialize({startOnLoad: true});</script>

        Actual
        ---------
        <p>Transforms mermaid blocks.</p>
        <div class="mermaid">graph TD;
            A--&gt;B;
            A--&gt;C;
            B--&gt;D;
            C--&gt;D;
        </div><pre><code class="language-mermaid">graph TD;
            A--&gt;B;
            A--&gt;C;
            B--&gt;D;
            C--&gt;D;
        </code></pre>
        <script src="mermaid.js"></script><script>mermaid.initialize({startOnLoad: true});</script>
```

**Comment by abhinav** [comment]
Thanks for the report! I'll have a fix up shortly.

**Comment by abhinav** [comment]
Fix released in v0.1.1

**Comment by rcrowe** [comment]
Works perfectly, thanks @abhinav 🙇🏻
